internal/cli/iam/users/users.go

```diff
@@ -29,5 +29,6 @@ func Builder() *cobra.Command {
 	}
 
 	cmd.AddCommand(InviteBuilder())
+	cmd.AddCommand(DescribeBuilder())
```

Comment:
I found out the other day we can do
```suggestion
	cmd.AddCommand(
		InviteBuilder(),
		DescribeBuilder(),
	)
```